src/main/scala/viper/gobra/util/ViperChopper.scala

```diff
@@ -69,7 +69,7 @@ object ViperChopper {
       var forallSmallNode: (Int => Boolean) => Boolean = null // for the safety check
 
       val result = {
-        if (nodes.size < 10) {
+        if (nodes.size <= 2) {
```

Comment:
Won't this make it very unlikely to ever execute the non-SCC algorithm? If that is the case, I wonder if we should keep this algorithm that only kicks in in small programs

Reply:
Yes, you are right. Thanks!